src/Operation/OperationInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Exo\Operation;
+
+interface OperationInterface
```

Comment:
Could we get by without the interface, simply type-hinting against the abstract class?

Reply:
Sure! Updates made to reflect this change.